coruscant/src/utils/firebase.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable prefer-destructuring */
+import * as firebase from 'firebase'
+import { externals } from '../../webpack.config'
+
+const config = externals.config
+
+const app = firebase.initializeApp(config.firebase)
```

Comment:
@KrashLeviathan I still think it's better to initialize this in the index.js, utils doesn't seem like the right place. thoughts?? @jackcmeyer

Reply:
I haven't looked through the app. But, if this file is only imported once, then we should be okay. However, I think it makes sense that this be done in the index .js file, since firebase should only be initialized once per application.

Reply:
Since this is a constant, doesn't it only get initialized once regardless of how many times it gets imported? My original solution was to keep it in index and do `window.firebaseInstance = firebase.initializeApp()`, but I didn't think globals were a good idea. I can still do that if you like, but we need to be able to access the instance returned by `firebase.initializeApp()` elsewhere in the application.